### PR TITLE
Ticket-tag: remove GET /ticket-tags

### LINKED ISSUE #4161: Ticket-tag: remove GET for /ticket-tags
Parent issue #4101.
Related issue: #4119.

Make `/ticket-tags` POST only.

### PR DESCRIPTION
**Pull request by enigmaeth**
Fixes #4161 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `nextgen` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

Remove GET for /ticket-tags.

### CODE REVIEW

File: app/api/ticket_tags.py [hunk 1]
```diff
@@ -65,32 +63,10 @@ def before_post(self, args, kwargs, data):
         if not has_access('is_coorganizer', event_id=data['event']):
             raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
 
-    def after_create_object(self, obj, data, view_kwargs):
```
Comment (poush) [inline app/api/ticket_tags.py]: is this is removed because JSON API taking care of filling this table?

Comment (enigmaeth) [inline app/api/ticket_tags.py]: Yes.

Comment (enigmaeth) [inline app/api/ticket_tags.py]: @poush Yes!

**Comment by shubham-padia** [comment]
@enigmaeth can you also add the get for endpoints like  `/events/id/ticket-tags` in this PR itself

**Comment by magdalenesuo** [comment]
@shubham-padia I think this is already done in previous pr and merged. [#4140](https://github.com/fossasia/open-event-orga-server/pull/4140/files#diff-f4c05149e2001f8002755304b9c14c80R111)
@enigmaeth Can GET for ticket tags not be for admins? Also, add in API doc for list GET from this `/events/id/ticket-tags` endpoint.

**Comment by enigmaeth** [comment]
@magdalenesuo @shubham-padia Added docs for GET `/events/id/ticket-tags` and `tickets/id/ticket-tags`.
GET /ticket-tags is not required.  @niranjan94 Can you confirm please?

**Comment by magdalenesuo** [comment]
@enigmaeth Can you check this out? [https://github.com/fossasia/open-event/blob/master/dev-handbook/resources-permissions.md](https://github.com/fossasia/open-event/blob/master/dev-handbook/resources-permissions.md). In that endpoint it does say admin can View?

**Pull request merged**